book/A-git-in-other-environments/sections/visualstudio.asc

```diff
@@ -1,7 +1,7 @@
 === Git in Visual Studio
 
 (((Visual Studio)))
-Visual Studio has Git tooling built directly into the IDE, starting with Visual Studio 2019 version 16.8.
+Visual Studio has Git tooling built directly into the IDE. On previous versions of Visual Studio, Git support can be found in Team Explorer. Starting from Visual Studio 2019 version 16.8, Visual Studio has an all new Git experience. We recommend using the latest version of Visual Studio to get the best Git experience.
```

Comment:
Sorry for the delay. The content here is fine, but can we split each sentence onto its own line? It helps with future diffs.
```suggestion
Visual Studio has Git tooling built directly into the IDE.
On previous versions of Visual Studio, Git support can be found in Team Explorer. 
Starting from Visual Studio 2019 version 16.8, Visual Studio has an all new Git experience. 
We recommend using the latest version of Visual Studio to get the best Git experience.
```